### PR TITLE
chore: skip prepack for gitpod

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

In gitpod, `prepack` is ensured in `init`, so we can skip this to accelerate the preparation for a dev preview.

Btw, this PR is created via Gitpod in a browser.

## Testing

https://gitpod.io/#https://github.com/logto-io/logto/pull/1883

### CODE REVIEW

**Comment by simeng-li** [comment]
> Btw, this PR is created via Gitpod in a browser.

Do they have Copilot?

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by darcyYe — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "bootstrap": "lerna bootstrap",
     "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
     "prepack": "lerna run --stream prepack",
-    "dev": "lerna run --stream prepack -- --incremental && lerna --ignore=@logto/integration-test run --parallel dev",
+    "dev:watch": "lerna --ignore=@logto/integration-test run --parallel dev",
```
Comment (charIeszhao) [inline package.json:12]: But why it doesn't use `watch` in this command?

Comment (wangsijie) [reply package.json]: removed this script name

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**